src/index.html

```diff
@@ -11,7 +11,138 @@
+            min="1" 
+            max="100" 
+            value="12" 
+          >
+        </label>
+        <label class="form-filed">
+          Full date of birth:
+          <input
+            type="date" 
+            name="date" 
+          >
+        </label>
+        <label class="form-filed">
+            I accept the term of the agreement
+          <input
+            type="checkbox" 
+            name="terms" 
+            required 
+          >
+        </label>
+      </fieldset>   
+      <fieldset class="form-group">
+        <legend>Registration</legend>
+        <label class="form-filed">
+          E-mail:
+          <input
+            type="email" 
+            name="email" 
+            multiple  
+            placeholder="email@example.com" 
+          >
+        </label>
+        <label class="form-filed">
+          Password:
+          <input
+            type="password" 
+            name="password" 
+            autocomplete="off" 
+            minlength="6"
+            maxlength="15"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-group">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-filed">
+          Do you love cats?
+          <input
+            type="radio" 
+            name="Yes" 
+          >
+            Yes
+          <input
+            type="radio" 
+            name="No" 
+          > 
+            No
+        </label>
+        <label class="form-filed">
+          What is your favorite color?
+          <input
+            type="color" 
+            name="color?"
+          >
+        </label>
+        <label class="form-filed">
+          What time do you go to bed?
+          <input
+            type="time" 
+            name="time"
+          >
+        </label>
+        <label class="form-filed">
+          What ere your favorite brand of cars?
+          <select name="cars"multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label> 
+        <label class="form-filed">
+          How do you rate our work?
+          <input
+            type="range" 
+            name="rating" 
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="form-group">
+        <legend>Additional info:</legend>
+        <label class="additional-info">
+            Comments:
+          <textarea 
+            name="comments"
+          >
+          </textarea>
+        </label>
+        <label class="form-filed">
+            Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+      <button type="submit">Submit</button>
+    </form>
     <script type="text/javascript" src="./main.js"></script>
   </body>
-</html>
+</html>
```

Comment:
Add a newline
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

Reply:
- [DEMO LINK](https://DmitriyChernega.github.io/layout_html-form/)
- [TEST REPORT LINK](https://DmitriyChernega.github.io/layout_html-form/report/html_report/)